aspnetcore/fundamentals/middleware/extensibility-third-party-container/sample/Startup.cs

```diff
@@ -0,0 +1,84 @@
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.DependencyInjection;
+using SimpleInjector;
+using SimpleInjector.Lifestyles;
+using MiddlewareExtensibilitySample.Data;
+using MiddlewareExtensibilitySample.Middleware;
+
+namespace MiddlewareExtensibilitySample
+{
+    public class Startup
+    {
+        private Container _container = new Container();
+
+        #region snippet1
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddMvc();
+
+            // Replace the default middleware factory with the 
+            // SimpleInjectorMiddlewareFactory. Pass in the 
+            // Simple Injector container when the factory is
+            // instantiated.
+            services.AddTransient<IMiddlewareFactory>(_ =>
+            {
+                return new SimpleInjectorMiddlewareFactory(_container);
+            });
+
+            // Wraps ASP.NET requests in an 
```

Comment:
Wraps the request in a [something] - not a type name.

Reply:
This is from the Simple Injector comments that summarize the purpose of this method. I'll see if I can do better on the next pass.